index.bs

```diff
@@ -476,8 +477,18 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
       <p>
         If present, the <dfn for='VideoConfiguration' dict-member>scalabilityMode</dfn>
         member represents the scalability mode as defined in [[webrtc-svc]].
+        Scalability mode is only applicable to the MediaEncodingType webrtc.
       </p>
 
+      <p>
+        If present, the <dfn for='VideoConfiguration' dict-member>spatialScalability</dfn>
+        member represents the ability to do spatial prediction, that is,
+        using frames of a resolution different than the current resolution as
+        dependencies. This is closely coupled to {{scalabilityMode}} in the
```

Comment:
I'd massage the phrasing of this sentence a bit for clarity. Something like:

... in the sense that streams encoded with modes using spatial scalability (e.g. "L2T1") can only be decoded if spatialScalability is supported.

Reply:
Done.